### ISSUE TITLE
composer installer name

### ISSUE THREAD

**Issue by silverstripesk** [bluehousegroup/silverstripe-contact-form#3]
Hi Aaron,

you should have installer-name in your composer file as "contact_form".

regards

pali

**Comment by sjdeere** [comment]
yeah how do I add this to my .json from  from my ssh command line.

normally i would do : `php composer  require  whatever/whatever` 

..but i when i try  bluehousegroup/silvertripe-contact-form  it doesn't recongise it.  could not find package
